package.json

```diff
@@ -11,8 +11,8 @@
     "tslint": "tslint -p tsconfig.json --format stylish",
     "eslint": "eslint --cache '**/*.ts?(x)'",
     "build": "node ./node_modules/typescript/bin/tsc -b .",
-    "test": "uvu -r ts-node/register --ignore dist",
-    "update-snapshots": "uvu -r ts-node/register --ignore dist --update-snapshots",
+    "test": "TS_NODE_TRANSPILE_ONLY=true uvu -r ts-node/register --ignore dist",
+    "update-snapshots": "TS_NODE_TRANSPILE_ONLY=true uvu -r ts-node/register --ignore dist --update-snapshots",
```

Comment:
Will this flag change skip type-checking of the test code? Or is that fine because the type-checking happens during the `build` step?

Reply:
> is that fine because the type-checking happens during the build step?

Correct. We can skip it here to speed up the command and allow type errors for local development.